Build fails with external qwtpolar

**Describe the bug**
qgis 3.18 fails to build with external qwtpolar
```
/usr/bin/ld: CMakeFiles/qgis_app.dir/gps/qgsgpsinformationwidget.cpp.o: in function `QgsGpsInformationWidget::QgsGpsInformationWidget(QgsMapCanvas*, QWidget*)':
qgsgpsinformationwidget.cpp:(.text+0x4c36): undefined reference to `QwtPolarPlot::QwtPolarPlot(QWidget*)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4c48): undefined reference to `QwtPolarPlot::setAutoReplot(bool)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4c6e): undefined reference to `QwtPolarPlot::setPlotBackground(QBrush const&)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4c9e): undefined reference to `QwtPolarPlot::setScale(int, double, double, double)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4cb3): undefined reference to `QwtPolarPlot::setAzimuthOrigin(double)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4cdd): undefined reference to `QwtPolarPlot::setScale(int, double, double, double)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4cf4): undefined reference to `QwtPolarGrid::QwtPolarGrid()'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4d0b): undefined reference to `QwtPolarGrid::setGridAttribute(QwtPolarGrid::GridAttribute, bool)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4d38): undefined reference to `QwtPolarGrid::setPen(QPen const&)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4d74): undefined reference to `QwtPolarGrid::setMinorGridPen(int, QPen const&)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4d89): undefined reference to `QwtPolarGrid::setMinorGridPen(int, QPen const&)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4d9d): undefined reference to `QwtPolarGrid::showAxis(int, bool)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4db1): undefined reference to `QwtPolarGrid::showAxis(int, bool)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4dc5): undefined reference to `QwtPolarGrid::showAxis(int, bool)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4dd9): undefined reference to `QwtPolarGrid::showAxis(int, bool)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4ded): undefined reference to `QwtPolarGrid::showAxis(int, bool)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4dfe): undefined reference to `QwtPolarGrid::showGrid(int, bool)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4e15): undefined reference to `QwtPolarGrid::showGrid(int, bool)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4e2c): undefined reference to `QwtPolarGrid::showMinorGrid(int, bool)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x4e40): undefined reference to `QwtPolarItem::attach(QwtPolarPlot*)'
/usr/bin/ld: CMakeFiles/qgis_app.dir/gps/qgsgpsinformationwidget.cpp.o: in function `QgsGpsInformationWidget::displayGPSInformation(QgsGpsInformation const&)':
qgsgpsinformationwidget.cpp:(.text+0x851d): undefined reference to `QwtPolarMarker::QwtPolarMarker()'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x854f): undefined reference to `QwtPolarMarker::setPosition(QwtPointPolar const&)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x8615): undefined reference to `QwtPolarMarker::setSymbol(QwtSymbol const*)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x8631): undefined reference to `QwtPolarMarker::setLabelAlignment(QFlags<Qt::AlignmentFlag>)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x8693): undefined reference to `QwtPolarMarker::setLabel(QwtText const&)'
/usr/bin/ld: qgsgpsinformationwidget.cpp:(.text+0x86a8): undefined reference to `QwtPolarItem::attach(QwtPolarPlot*)'
collect2: error: ld returned 1 exit status
```

Caused by the removal of ${QWTPOLAR_LIBRARY} from target_link_libraries in 93a3c6b987e9cda4dbe8dcede90c3211ad1aaa46

--- a/src/app/CMakeLists.txt
+++ b/src/app/CMakeLists.txt
@@ -510,6 +510,7 @@ add_dependencies(qgis_gui ui)
 
 target_link_libraries(qgis_app
   ${QWT_LIBRARY}
+  ${QWTPOLAR_LIBRARY}
   ${Qt5Sql_LIBRARIES}
   ${Qt5UiTools_LIBRARIES}
   ${OPTIONAL_QTWEBKIT}

